src/_vendor/gha_runner/gh.py

```diff
@@ -206,24 +206,30 @@ def get_runners(self) -> list[SelfHostedRunner] | None:
             an error in the request or the response is not a mapping object.
         """
         runners = []
-        try:
-            res = self.get(f"repos/{self.repo}/actions/runners")
-            # This allows for arbitrary mappable objects to be used
-            if not isinstance(res, collections.abc.Mapping):
-                # This could be related to the API or the request itself.
-                # ie the response is not a JSON object
-                raise RunnerListError(f"Did not receive mapping object: {res}")
-            for runner in res["runners"]:
-                id = runner["id"]
-                name = runner["name"]
-                os = runner["os"]
-                labels = [label["name"] for label in runner["labels"]]
-                runners.append(SelfHostedRunner(id, name, os, labels))
-            return runners if len(runners) > 0 else None
-        except RuntimeError as e:
-            # This occurs when we receive a status code is > 400
-            raise RunnerListError(f"Error getting runners: {e}")
-            # Other exceptions are bubbled up to the caller
+        per_page = 1
```

Comment:
Am I missing something or should this be like 20 or 5?

Reply:
Yeah, in a subsequent commit I get rid of that entirely and just rely on the default. This was here so I could test on my account, where there are only 2 zombie workers.

Reply:
Oh no I lied, I ended up keeping it for the next time someone has to test this, but setting it to 30, the API default.